### ISSUE TITLE
Usage?

### ISSUE THREAD

**Issue by ArieLeo** [DavidArayan/UnityDecalEditor#1]
Any chance to get a small guide on how to use the decal system? i'm testing it on unity 2017.2 at the moment and i cannot make it works

**Comment by DavidArayan** [comment]
Greetings,

Thanks for reaching out. Unfortunately I have not updated this code in over 2 years! so it's unlikely to work with the new unity version. This was made mostly as an experiment.

Fortunately all algorithms are written in C# and are platform independent so you're more than welcome to study/stitch them together for your own solution.

I haven't seen a large demand for open-source decal frameworks for unity since there are alot of great solutions in the asset store. I'd suggest having a look there for more complete and up to date solutions.

keep programming!

**Comment by ArieLeo** [comment]
Oh you might be surprised on how much people were hoping to find a good open source mesh decal in unity (Where people can collaborate and adding stuff).
And that is one of the reason why i'm testing a bunch of decal repo, most that i found are mostly abandoned or not support decal for terrain. I'm actually trying to find any opensource decal that support terrain, and might able to help developing it in my free time.
But no worries, and really thanks for replying a message for an old repo, really appreciate that :)

**Comment by DavidArayan** [comment]
Yeah, when I developed this close to 3 years ago it failed to grab any traction from the community. This was at the time when decal frameworks were popping up in the app store left-right and center. I have gotten a few requests recently to revive the project.

I'm in the middle of re-writing/modernising the core parts of another project "EzySlice" which actually shares the core fundamentals with this project (all the math/intersection code is essentially the same). It's a slow progress due to full time work however i'll get it done eventually. I'll see about bringing these projects back to life and see if the community starts contributing!

**Comment by ArieLeo** [comment]
No worries, take your time man.
